Samples~/AvatarCreatorSamples/AvatarCreatorElements/Scripts/SimpleAvatarCreator.cs

```diff
@@ -14,9 +14,13 @@ namespace ReadyPlayerMe.Samples.AvatarCreatorElements
     /// <summary>
     ///     A class responsible for creating and customizing avatars using asset and color selections.
     /// </summary>
+    [RequireComponent(typeof(UserAccountManager))]
     public class SimpleAvatarCreator : MonoBehaviour
     {
-        public UnityEvent<AvatarProperties> onAvatarCreated;
+        public UnityEvent<AvatarProperties> OnAvatarCreated;
+        public UnityEvent OnAvatarSaved;
```

Comment:
OnAvatarSaved and On AvatarSelected are basically doing the same thing, are firing an event when the avatar creation is finished. I think having to separate events here is redundant and a little confusing for developers
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/3489a4ed-8385-46e3-985b-8d0f6c81cb68)
I think it would make more sense to have some sort of AvatarCreationFinished event that fires in both cases (when either you click save or if you choose to use an existing avatar. And perhaps the finished event might even pass the gameobject so that users can use it EG applying animations 

And on that note I think that probably the OnAvatarCreated event should be renamed to OnAvatarLoaded I get that Created vs loaded is quite similar but to me Loaded makes more sense. This isn't really related to this PR though.